### ISSUE TITLE
qwen2-vl-7b-instruct模型加载报错

### ISSUE THREAD

**Issue by SharonJin422** [QwenLM/Qwen2-VL#148]
File /opt/conda/lib/python3.10/site-packages/transformers-4.45.0.dev0-py3.10.egg/transformers/modeling_utils.py:549, in load_state_dict(checkpoint_file, is_quantized)
    544 """
    545 Reads a PyTorch checkpoint file, returning properly formatted errors if they arise.
    546 """
    547 if checkpoint_file.endswith(".safetensors") and is_safetensors_available():
    548     # Check format of the archive
--> 549     with safe_open(checkpoint_file, framework="pt") as f:
    550         metadata = f.metadata()
    551     if metadata.get("format") not in ["pt", "tf", "flax", "mlx"]:

SafetensorError: Error while deserializing header: InvalidHeaderDeserialization

**Comment by fyabc** [comment]
@SharonJin422 Hi, the checkpoint you downloaded appears to be broken or corrupted. Please delete it and download again.

**Comment by SharonJin422** [comment]
hi, thanks, it's already solved.
And whether qwen2-vl-7b-instruct supports multiple rounds of conversations?

**Comment by fyabc** [comment]
> hi, thanks, it's already solved. And whether qwen2-vl-7b-instruct supports multiple rounds of conversations?

@SharonJin422 Yes, you can pass multiple turns of conversation messages directly to the model.
